tests/unit/snow/test_dialect_utils.py

```diff
@@ -16,6 +16,64 @@ def test_snow_query_with_no_unsupported_lca_usage():
     assert not error
 
 
+def test_snow_query_with_valid_alias_usage():
```

Comment:
can you remove "snow", since LCA will be generalized for all dialects.

```suggestion
def test_query_with_valid_alias_usage():
```

Reply:
Done